src/components/expanded-state/UniqueTokenExpandedState.tsx

```diff
@@ -263,7 +263,11 @@ const UniqueTokenExpandedState = ({
   const isNFT = uniqueTokenType === UniqueTokenType.NFT;
 
   // Fetch the ENS profile if the unique token is an ENS name.
-  const cleanENSName = isENS && uniqueId ? uniqueId?.split(' ')?.[0] : uniqueId;
+  const cleanENSName = isENS
+    ? uniqueId
+      ? uniqueId?.split(' ')?.[0]
+      : uniqueId
+    : '';
   const ensProfile = useENSProfile(cleanENSName, { enabled: isENS });
```

Comment:
thought `enabled: isENS` would have guarded against this?

Reply:
It is actually line 276, `prefetchENSProfileRecords({ name: cleanENSName });`, that causes the error. Currently cleanENSName can still be a truthy value even if even the asset isn't an ENS, so the prefetch will get triggered

Reply:
gotchaaa